### PR TITLE
Add Payback as medium

### PR DESCRIPTION
**Pull request by ConfusedAlex**

### CODE REVIEW

**Comment by ConfusedAlex** [comment]
Oops, I messed up the formatting. Will fix later

File: _data/sites.json [hunk 1]
```diff
@@ -15975,6 +15975,14 @@
         ]
     },
 
+    {
+       "name": "Payback",
+        "url": "https://www.payback.de/info/mein-payback/kontakt",
+        "difficulty": "medium",
```
Comment (tupaschoal) [inline _data/sites.json:15981]:
```suggestion
        "difficulty": "hard",
```
If it requires contacting, we put it as hard.

File: _data/sites.json [hunk 2]
```diff
@@ -15975,6 +15975,14 @@
...
+        "notes": "According to their help page, you need to contanct their Service-Center",
+        "domains": ["payback.de"]
```
Comment (tupaschoal) [inline _data/sites.json:15983]:
```suggestion
        "domains": [
            "payback.de"
        ]
```

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**